### PR TITLE
Secure/fix broken URLs (3 of 4)

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/rubberband.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Rubberband < Formula
   head "https://bitbucket.org/breakfastquay/rubberband/", :using => :hg
 
   stable do
-    url "http://code.breakfastquay.com/attachments/download/34/rubberband-1.8.1.tar.bz2"
```
Comment (ilovezfs) [inline Formula/rubberband.rb]: This seems to be the canonical url.

Comment (ilovezfs) [reply Formula/rubberband.rb]: Oh I see the download is missing. Probably should let them know:

https://bitbucket.org/breakfastquay/rubberband/issues

Comment (vszakats) [reply Formula/rubberband.rb]: It's the download link from the homepage, but the 'files' area seems to be misconfigured and I couldn't find it anywhere else either. The link also looks on the "accidental" side to be worthy noting in a comment and easy to lookup on the homepage, so I haven't made a copy-paste of it.

**Comment by vszakats** [comment]
@BrewTestBot test this please

File: Formula/rubberband.rb (hunk 1 above)
Comment (vszakats) [reply Formula/rubberband.rb]: Upstream report: https://bitbucket.org/breakfastquay/rubberband/issues/21/

**Comment by vszakats** [comment]
All errors were due to sourceforge.net [ups and downs](https://twitter.com/sfnet_ops/status/967960888027373568) yesterday.

**Pull request merged**